Suppress warnings and bump dependency Gems

singhprd opened the pull request:
Hi there Gareth!

I've been using this Gem for a little project of mine, and thought I'd dive in and see if I could improve anything.

I've forked the repo, but if you want, merge in my changes 😄 

This PR is super simple and just suppresses warnings when running the tests, as well as bumping the versions of Gems.

Thanks 👍

singhprd commented:
The warnings were just about declared but unused variables, and one from 
https://github.com/bblimke/webmock/commit/0bfe6784d0e00e9cc4b404ca1dffeabc0fe201d2

gareth commented:
Hi Peter!

Glad the code's still useful 4 years from when I wrote it 😀. I'm happy to include these updates but it's probably better not to include version bumping in a PR. It makes it harder to combine other updates in the same version release - for example I notice the API host has changed and needs updating too, and I'll probably make that change at the same time

If you could edit your first commit to remove the version bump then I will definitely merge this 👍

singhprd committed:
Removed unused variables from tests and updated Gems in Gemfile

singhprd commented:
Hi again Gareth,

Yeah its defiantly still useful! It got me up and running with the API quickly, which was really nice, so thanks for the code and for releasing it to the wold 😄 

I've now cleaned up my changes into a single commit and removed the version bumping - thanks for the pointer here 👍 

Good spot on the API host front - it still works but defiantly worth changing just incase they turn it off in the future.

Cheers!
